rotkehlchen/exchanges/binance.py

```diff
@@ -680,6 +681,73 @@ def _query_margined_futures_balances(
 
         return balances
 
+    def _query_pools_balances(
+            self,
+            balances: DefaultDict[Asset, Balance],
+    ) -> DefaultDict[Asset, Balance]:
+
+        def process_pool_asset(asset_name: str, asset_amount: FVal) -> None:
+
+            if asset_amount == ZERO:
+                return None
+
+            try:
+                asset = asset_from_binance(asset_name)
+            except UnsupportedAsset as e:
+                self.msg_aggregator.add_warning(
+                    f'Found unsupported {self.name} asset {asset_name}. '
+                    f'Ignoring its margined futures balance query. {str(e)}',
+                )
+                return None
+            except UnknownAsset as e:
+                self.msg_aggregator.add_warning(
+                    f'Found unknown {self.name} asset {asset_name}. '
+                    f'Ignoring its margined futures balance query. {str(e)}',
+                )
+                return None
+            except DeserializationError as e:
+                log.error(
+                    f'{self.name} balance deserialization error ',
+                    f'with asset {asset_name}. {str(e)}',
+                )
+                return None
+
+            try:
+                usd_price = Inquirer().find_usd_price(asset)
+            except RemoteError as e:
+                self.msg_aggregator.add_error(
+                    f'Error processing {self.name} balance entry due to inability to '
+                    f'query USD price: {str(e)}. Skipping margined futures balance entry',
+                )
+                return None
+
+            balances[asset] += Balance(
+                amount=asset_amount,
+                usd_value=asset_amount * usd_price,
+            )
+
+            return None
```

Comment:
nitpick: Remove unneeded newline

```suggestion
            )
            return None
```